scm/scm_manager.go

```diff
@@ -0,0 +1,115 @@
+package scm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/beatlabs/github-auth/app"
+	"github.com/beatlabs/github-auth/key"
+	"github.com/google/go-github/v45/github"
+	"go.uber.org/zap"
+)
+
+const installationAPI = "https://api.github.com/app/installations"
+
+// GithubConfig keeps parameters of the GitHub app.
+type GithubConfig struct {
```

Comment:
After more careful checking, it seems this struct only needs `appID` and `appConfig`; all the other fields are not used.